usvm-ts/src/main/kotlin/org/usvm/TSBinaryOperator.kt

```diff
@@ -6,32 +6,93 @@ sealed class TSBinaryOperator(
     val onBool: TSContext.(UExpr<UBoolSort>, UExpr<UBoolSort>) -> UExpr<out USort> = shouldNotBeCalled,
     val onBv: TSContext.(UExpr<UBvSort>, UExpr<UBvSort>) -> UExpr<out USort> = shouldNotBeCalled,
     val onFp: TSContext.(UExpr<UFpSort>, UExpr<UFpSort>) -> UExpr<out USort> = shouldNotBeCalled,
+    val onRef: TSContext.(UExpr<UAddressSort>, UExpr<UAddressSort>) -> UExpr<out USort> = shouldNotBeCalled,
```

Comment:
Please, add comments about desired sorts and ban sorts here, it is not obvious

Reply:
Done